features/object-js.yml

```diff
@@ -0,0 +1,44 @@
+name: Object
```

Comment:
Surely this should be `[object Object]`, with the filename `object-object.yml`? 😂

I'm only joking. Mostly.